app/javascript/components/map/map-actions.js

```diff
@@ -4,7 +4,10 @@ import { fetchLayerSpec } from 'services/layer-spec';
 
 const setLayerSpecLoading = createAction('setLayerSpecLoading');
 const setLayerSpec = createAction('setLayerSpec');
-const setLayers = createAction('setLayers');
+const setMapOptions = createAction('setMapOptions');
+const setMapZoom = createAction('setMapZoom');
+const mapZoomIn = createAction('mapZoomIn');
```

Comment:
I think you achieve this with a single action. changeMapZoom = (change) => state.map.zoom + change; and then set the buttons to pass +1 or -1? That way we can handle cases where we want to zoom by more than 1 step in the future.

Reply:
nice feedback! makes sense